connect/connectServer

```diff
@@ -268,6 +268,13 @@ function handleRequest(request, response) {
       }
     });
     request.on('end', function() {
+      // No ID cookie, the user has logged out.
+      if (!/(^|;)\s*ID=\w/.test(request.headers.cookie)) {
```

Comment:
I'm glad of the comment.  Could the check itself be made any easier to understand?  It would be lovely if it could read something like `if(!cookies.has('ID')) {`… (presuming that `request.headers.cookie` has been split out into a Map named `cookies`).